Gemfile

```diff
@@ -31,7 +37,6 @@ end
 group :system_tests do
   gem 'beaker'
   gem 'beaker-rspec'
-  gem 'beaker-windows'
   gem 'simp-beaker-helpers', ENV['SIMP_BEAKER_HELPERS_VERSION'] || ['>= 1.18.7', '< 2']
```

Comment:
beaker-windows does not look like it should have been removed

Reply:
I've seen cases where you need it and cases where it raises a warning. I'm adding it back because I have no idea when/why these cases occur.

Reply:
OIC...it's in `Gemfile.project` :-|